README.rst

```diff
@@ -75,7 +75,7 @@ PostgreSQL is used as the database backend for Insights-rbac. A docker-compose f
 
 Assuming the default .env file values are used, to access the database directly using psql run ::
 
-    psql rbac -U rbacadmin -h localhost -p 15432
+    psql postgres -U postgres -h localhost -p 15432
```

Comment:
default user/pass was `postgres` for me locally, so updated this. afaik they are the default but if not ill revert

Reply:
Thanks for updating this ^ i've been meaning to fix that 😂